packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -814,6 +815,11 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 					editor.addShortcut('Meta+Shift+7', '', () => editor.execCommand('InsertOrderedList'));
 					editor.addShortcut('Meta+Shift+8', '', () => editor.execCommand('InsertUnorderedList'));
 					editor.addShortcut('Meta+Shift+9', '', () => editor.execCommand('InsertJoplinChecklist'));
+					editor.addShortcut(
+						KeymapService.instance().getAccelerator('focusToolbar'),
+						'Focus toolbar',
```

Comment:
What is the purpose of the "Focus toolbar" string here? Can it be omitted like in the other addShortcut calls? (especially since it's not translated)